### ISSUE TITLE
[BUG]: Charts don't load in Linux Ubuntu versions

### ISSUE THREAD

**Issue by leocan3** [KomodoPlatform/atomicDEX-Desktop#2266]
**Describe the bug**

Using either appimage or portable versions on Ubuntu 22.04 the charts never load, there's always a spinning wheel with "Loading market data...". Other than that Adex works normally. 

**Operating Environment(s):**
 - OS: Ubuntu 22.04.2
 - AtomicDEX Version: 0.5.7.2 and older ones

**Additional context**
This bug happened to me also with the older versions that used nomics charts and the same is happening with the latest one using livecoinwatch charts

I tried on 3 different computers behind the same home router: 2 with Ubuntu and 1 with Windows. The result is that the Windows version works: all the charts load without problems. On the contrary both Ubuntu computers don't load the charts. 

One of the Ubuntu computer has a dedicated NVIDIA RTX 2070 graphic card, the other just embedded graphics. This makes no difference.

I tried also in a brand new Ubuntu install in virtualbox and again the charts don't load.

Also one of the Ubuntu computers is dual-boot, if I boot in Windows and install the win version the charts load normally.

So in conclusion:
1. It isn't related to my home network
2. It isn't related to a specific chart system (neither nomics or livecoincharts load)
3. It isn't related to a specific hardware
4. It only happens with linux versions on Ubuntu, while windows versions work

**Comment by ShorelineCrypto** [comment]
I can report that the latest version 0.7.2-beta chart does not work in MX Linux (debian based) version 23.3 which runs on Debian 12 (bookworm).  The ubuntu AppImage file can run directly, but chart issue is exactly as above said, spinning wheel.

**Comment by BlackcoinDev** [comment]
I can confirm with 0.8.0 latest RC same issue occurs with Appimage on Ubuntu 22.04 and Fedora 39 and Fedora 40.

Running Appimage (Debug) from https://github.com/KomodoPlatform/komodo-wallet-desktop/actions/runs/10580881553
